src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
-'use strict';
+const logo = document.querySelector('.logo');
 
-// write your code here
+function createResolvedPromise(message) {
+  return new Promise((resolve, reject) => {
+    resolve(message);
+  });
+}
+
+function createRejectedPromise(message, timeout) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(message);
+    }, timeout);
+  });
+}
```

Comment:
The `createRejectedPromise` function correctly uses a timeout to reject the promise. Ensure that the `timeout` parameter is passed as a number representing milliseconds.